docker/crontab

```diff
@@ -1,5 +1,5 @@
 # Compute stats
-5 * * * * root /usr/local/bin/python /code/manage.py compute_stats >> /var/log/stats_calc.log 2>&1
+5 * * * * root /usr/local/bin/python /code/manage.py compute_stats
```

Comment:
do we know if this is going to do what we want? If cron is running as a job in runit, the output of these cron jobs isn't going to go to the container's stdout. I don't know how to make this do what we want it to do in this case. Thoughts?